pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
   "typing_extensions",
   "pydantic",
   "databricks-sdk>=0.34.0",
+  "databricks-vectorsearch>=0.50",
```

Comment:
If we are adding this here, can we remove it from other project deps as well so we need to update only in one place rather than multiple projects?